src/index.html

```diff
@@ -7,10 +7,31 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
+    <title>Search bars v1.4</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="wrapper">
+      <form data-qa="big">
+        <input
+        type="search"
+        name="bigBar"
+        id="searchBarBig"
+        class="bar bigBar"
+        placeholder="Try &#34Los Angeles&#34"
+        data-qa="keypress"
+      >
```

Comment:
```suggestion
      <input
        type="search"
        name="bigBar"
        id="searchBarBig"
        class="bar bigBar"
        placeholder="Try &#34Los Angeles&#34"
        data-qa="keypress"
      >
```
Форматування поплило трохи

Reply:
Перевірив усе, працює як задумано
Дякую за підказку щодо background :)